l10n_do_accounting/migrations/13.0.1.0.1/pre-init_document_type.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from odoo import api, SUPERUSER_ID
+
+_logger = logging.getLogger(__name__)
+
+
+def update_document_types_not_vat_required(env):
+    query = """
+    UPDATE l10n_latam_document_type
+    SET is_vat_required = 'f'
+    WHERE is_vat_required = 't'
+    AND country_id = {company}
```

Comment:
company=env.ref("base.do").id   
....
AND country_id = {company} 👀

Reply:
Era pa ve